docs/_docs/reference/changed-features/structural-types.md

```diff
@@ -131,16 +128,14 @@ of `reflectiveSelectable` shown above. What happens "under the hood" is then the
      reflectiveSelectable(f).applyDynamic("close")()
```

Comment:
it might be strange that this is the only place where we suddenly introduce `applyDynamic`, previously it looked like there was only `selectDynamic`

Reply:
I think it's acceptable, as the example requires it, and `applyDynamic` is introduced as early as line 84